src/components/neumorphic/LightSourceSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { MouseEvent } from 'react';
+import style from '../../style/sass/components/neumorphic/configuration.module.scss';
+
+interface LightSourceSelectorProps {
+  lightSource: number;
+  disabled?: boolean;
+  onDirectionChanged: (name: number) => void;
+}
+
+const LightSourceSelector = ({
+  lightSource,
+  onDirectionChanged,
+  disabled,
+}: LightSourceSelectorProps) => {
+  const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+    const direction = Number((event.target as HTMLInputElement).name);
+    onDirectionChanged(direction);
+  };
+  return (
+    <div className={`${style.row} ${style.label}`} style={{ margin: '12px 0px 8px' }}>
```

Comment:
Evita estos estilos de linea si puedes y muevelo a su respectiva clase en los css modules
```suggestion
    <div className={`${style.row} ${style.label}`}>
```